src/commands.cpp

```diff
@@ -2688,6 +2688,35 @@ void CommandDelUA::procresult()
 
 #endif
 
+
+
+CommandGetUserEmail::CommandGetUserEmail(MegaClient *client, const char *uid)
+{
+    cmd("uge");
```

Comment:
Agree! Actually, most of the commands do not set `this->client`, and the right place would be the `Command` ctor. We can plan to amend that for all the commands at once.